pkgs/development/compilers/llvm/11/default.nix

```
@@ -6,16 +6,18 @@
 }:
 
 let
-  release_version = "11.0.0";
-  version = release_version; # differentiating these (variables) is important for RCs
+  release_version = "11.0.1";
+  candidate = ""; # empty or "rcN"
+  dash-candidate = stdenv.lib.optionalString (builtins.stringLength candidate != 0) "-${candidate}";
```

```suggestion
  dash-candidate = stdenv.lib.optionalString (candidate != "") "-${candidate}";
```

Shouldn't make a difference but is a bit more compact (nitpick).